scss/_utilities.scss

```diff
@@ -361,6 +361,16 @@ $utilities: map-merge(
         reset: inherit,
       )
     ),
+    "line-height": (
+      property: line-height,
+      class: lh,
+      values: (
```

Comment:
Maybe we use a similar numerical scale as our margin/padding utilities?

Reply:
I've been thinking about that, but I'm not sure what values to use here. `lh-1` would make sense for a line height of `1`, but what could we use for the other values?

Reply:
Yeah, it'd be too weird to do anything numerical. Maybe something else using similar vocabulary from the rest of the project?

- `lh-1`
- `lh-compact`
- `lh-base`
- `lh-spacious`

But I think `sm`, `base`, and `lg` are best.